appveyor.yml

```diff
@@ -25,6 +25,10 @@ before_build:
   - cmd: dotnet --version
   # Display minimal restore text
   - cmd: dotnet restore --verbosity m
+  # Prepare localdb
+  - cmd: SqlLocalDB.exe create MSSQLLocalDB
```

Comment:
I cannot see why this is necessary,

Reply:
The build crashed several times by time-out because of 
a local database did not have time to be created. https://ci.appveyor.com/project/Tazmainiandevil/tablestorage-abstractions/build/2.0.54